dpnp/dpnp_iface_manipulation.py

```diff
@@ -967,6 +969,44 @@ def broadcast_arrays(*args, subok=False):
     return [dpnp_array._create_from_usm_ndarray(a) for a in usm_arrays]
 
 
+def broadcast_shapes(*args):
+    """
+    Broadcast the input shapes into a single shape.
+
+    For full documentation refer to :obj:`numpy.broadcast_shapes`.
+
+    Parameters
+    ----------
+    *args : tuples of ints, or ints
+        The shapes to be broadcast against each other.
+
+    Returns
+    -------
+    tuple
+        Broadcasted shape.
+
+    See Also
+    --------
+    :obj:`dpnp.broadcast_arrays` : Broadcast any number of arrays against
+                                   each other.
+    :obj:`dpnp.broadcast_to` : Broadcast an array to a new shape.
+
+    Examples
+    --------
+    >>> import dpnp as np
+    >>> np.broadcast_shapes((1, 2), (3, 1), (3, 2))
+    (3, 2)
+    >>> np.broadcast_shapes((6, 7), (5, 6, 1), (7,), (5, 1, 7))
+    (5, 6, 7)
+
+    """
+
+    if hasattr(numpy, "broadcast_shapes"):
+        return numpy.broadcast_shapes(*args)
+
+    return _broadcast_shapes(*args)
```

Comment:
`_broadcast_shapes` requires a list of array as an input, which is not the case here.
We should either replace it with `_broadcast_shape_impl(args)` from dpctl or use `numpy.broadcast_shapes` only, which is also an option, because it is available since 1.20 version (which is EOL already and so any actual older numpy version has it).